t/app/helpers/send_email.t

```diff
@@ -42,7 +42,7 @@ my $email = Email::MIME->new($email_as_string);
 my $expected_email_content =   path(__FILE__)->parent->child('send_email_sample.txt')->slurp;
 my $name = FixMyStreet->config('CONTACT_NAME');
 $name = "\"$name\"" if $name =~ / /;
-my $sender = $name . ' <' . FixMyStreet->config('DO_NOT_REPLY_EMAIL') . '>';
+my $sender = '"' . $name . '" <' . FixMyStreet->config('DO_NOT_REPLY_EMAIL') . '>';
```

Comment:
Is this duplicating the effort of line 44? For example if `CONTACT_NAME` is `Foo Bar`, will you end up with `""Foo Bar""` here?